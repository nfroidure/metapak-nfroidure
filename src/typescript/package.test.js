'use strict';

const packageTransformer = require('./package');

describe('private', () => {
  describe('Package transformer', () => {
    it('should work', () => {
      expect(
        packageTransformer({
          types: 'index.d.ts',
          metapak: {
            data: {},
          },
        })
      ).toMatchSnapshot();
    });
  });
});
