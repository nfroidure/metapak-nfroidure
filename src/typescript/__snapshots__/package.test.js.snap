// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`private Package transformer should work 1`] = `
Object {
  "devDependencies": Object {
    "rimraf": "^3.0.1",
    "typescript": "^3.7.5",
  },
  "greenkeeper": Object {
    "ignore": Array [
      "typescript",
    ],
  },
  "metapak": Object {
    "data": Object {},
  },
  "scripts": Object {
    "precz": "npm run types",
    "preversion": "npm run types",
    "types": "rimraf -f 'dist/**/*.d.ts' && tsc --project . --declaration --emitDeclarationOnly --outDir dist",
  },
  "types": "index.d.ts",
}
`;
